docs/sources/developer/Developing-MacOS.md

```diff
@@ -0,0 +1,45 @@
+# Developing on MacOS
+
+## Git
+To download the latest version of Plaso you'll need to install the 
```

Comment:
likely easier to get xcode command line utils? If I'm not mistaken that contains git

Reply:
Probably. I'd like to update this whole section later.

Reply:
ack